report/03_methodology/03_database_management.md

```diff
@@ -0,0 +1,34 @@
+## Database management
+
+The soTired application is divided into two areas, the stand-alone client side as well as the server side with options for evaluation and export. 
+In order to make both areas independent of each other and also to ensure that the values are always available, even if there is no network connection at a certain point of use, it was agreed to create and manage an independet database on both sides.
+
+### Client database
+
+As described above, the client side is written with the help of the cross platform framework *Flutter* and the programming language *dart*. 
+As client side database *Hive* comes in handy.
+Hive describes itself as a 'Fast, Enjoyable & Secure NoSQL Database' [5]. 
+That makes it easy to store database objects in boxes and access them again. 
+Hive in conjunction with Flutter also promises great support in terms of use, as this has been specially adapted to it [5]. 
+
+In addition, a NoSQL database was chosen because it is more flexible in terms of the variety of data in the area of research results and the speed required for storing such a large amount of data. 
```

Comment:
```suggestion
In addition to *Hive*, a NoSQL database was chosen because it is more flexible in terms of the variety of data in the area of research results and the speed required for storing such a large amount of data. 
```